cmd/kubeadm/app/util/config/cluster_test.go

```diff
@@ -364,7 +401,34 @@ func TestGetAPIEndpoint(t *testing.T) {
 		expectedError bool
 	}{
 		{
-			name: "valid",
+			name: "valid v1beta1",
+			configMap: fakeConfigMap{
+				name: kubeadmconstants.InitConfigurationConfigMap, // ClusterConfiguration from kubeadm-config.
+				data: map[string]string{
+					kubeadmconstants.ClusterStatusConfigMapKey: string(cfgFiles["ClusterStatus_v1beta1"]),
+				},
+			},
+		},
+		{
+			name: "invalid v1beta1 - No CLusterStatus in kubeadm-config ConfigMap",
```

Comment:
super minor nit:
`CLusterStatus` -> `ClusterStatus` 
(appears more than once in this file)